README.md

```diff
@@ -133,7 +133,7 @@
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](https://github.com/patriciachrysy/hello-world.git).
+Feel free to check the [issues page](https://github.com/patriciachrysy/hello-world.git/issues).
 
```

Comment:
- [ ] As raise earlier, the issue link in the README file still redirected to your project instead of the issues page. Kindly look through the url carefully and make neccessary adjustment. ℹ️ 

```suggestion
https://github.com/patriciachrysy/hello-world/issues
```